addons/web/static/src/legacy/js/core/dialog.js

```diff
@@ -238,6 +238,13 @@ var Dialog = Widget.extend({
      *   `on_close` handler.
```

Comment:
I was confuse with this : https://www.gitkraken.com/learn/git/best-practices/git-commit-message#git-commit-message-structure :)
According to @rdeodoo this is more likely an IMP because there is no traceback.
I'll add some steps to reproduce the flow. Thanks.